specs/metricbeat.yml

```diff
@@ -0,0 +1,161 @@
+version: 2
+inputs:
+  - name: beat/metrics
+    description: "Beat metrics"
+    platforms: &platforms
+      - linux/386
+      - linux/amd64
+      - linux/arm64
+      - linux/ppc64le
+      - darwin/amd64
+      - darwin/arm64
+      - windows/amd64
+      - windows/arm64
+      - container/386
+      - container/amd64
+      - container/arm64
+    outputs: &outputs
```

Comment:
It's sad we cannot include file so we can common reusable, but still explicit is better.